packages/issue-receiver/src/main.ts

```diff
@@ -0,0 +1,48 @@
+import * as core from '@actions/core'
+import * as gh from '@actions/github'
+import * as Webhooks from '@octokit/webhooks'
+
+// TODO: Generalise this to work on any object
+interface Notification {
+    'list-ops': Exercise
+}
+
+interface Exercise {
+    commit_sha: string
+    commit_message: string
+    pull_request_html_url: string
```

Comment:
```suggestion
    pull_request_html_url?: string
```

We have to infer these from the commit message, which may not always be possible.